deploy/charts/cert-manager/templates/psp-clusterrole.yaml

```diff
@@ -0,0 +1,17 @@
+{{- if .Values.global.podSecurityPolicy.enabled }}
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: {{ template "cert-manager.fullname" . }}-psp
+  labels:
+    app: {{ include "cert-manager.name" . }}
+    chart: {{ include "cert-manager.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+rules:
+- apiGroups: ['policies']
```

Comment:
This should be `policy` I think, as the API group name is `policy` and apiVersion is `policy/v1beta1`. We don't pluralise groupNames 😄